styles.css

```diff
@@ -136,3 +140,294 @@ body {
 .main__button:hover {
   opacity: .5;
 }
+
+@media (min-width: 1600px) {
+  .wrapper {
+    margin: auto;
+    width: 80%;
+  }
+
+  .header {
+    margin-bottom: 9%;
+  }
+
+  .menu__item {
+    margin-right: 30px;
+    font: 500 18px/20px "Roboto", arial, helvetica, sans-serif;
+  }
+
+  .main {
+    width: 37%;
+    padding-right: 63%;
+    background-size: 760px;
+    background-position: top right;
+    padding-top: 4%;
+    padding-bottom: 13%;
+    background-position-x: 99%;
+  }
+
+  .main__title {
+    margin: 0 0 40px 0;
+    font: bold 66px/87px "pt_serif", arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin: 0 0 56px 0;
+    font: 500 22px/26px "Roboto", arial, helvetica, sans-serif;
+    max-width: 455px;
+  }
+
+  .main__button {
+    font: 500 19px/24px "Roboto", arial, helvetica, sans-serif;
+    padding: 18px 22px;
+  }
+}
+
+@media (max-width: 1360px) {
+    .wrapper {
+      margin: auto;
+      width: 91%;
+    }
+  }
+
+  @media (max-width: 1000px) {
+    .wrapper {
+      margin: auto;
+      width: 90%;
+    }
+
+    .header {
+      margin-bottom: 9%;
+    }
+
+    .menu__item {
+      margin-right: 30px;
+      font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+    }
+
+    .main {
+      width: 40%;
+      padding-right: 60%;
+      background-size: 300px;
+      background-position: top right;
+      padding-top: 4%;
+      padding-bottom: 13%;
+      background-position-x: 99%;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6356170)